src/status_im/common/floating_button_page/view.cljs

```diff
@@ -52,7 +52,8 @@
      :remove-listeners    remove-listeners}))
 
 (defn view
-  [{:keys [header footer customization-color gradient-cover?]} &
+  [{:keys [header footer customization-color footer-container-padding gradient-cover?]
+    :or   {footer-container-padding (safe-area/get-top)}} &
```

Comment:
I've seen  you updated the usages to pass `0` to `footer-container-padding`.

So I'm wondering if we actually need this value or not. if not, can we just remove this prop then?

Reply:
If keyboard shows autocorrect view (default behavior for inputs), it needs an extra padding, so we still need this.

Reply:
Incredible we have to manually handle that part ._.

thanks for the explanation! :+1: